Gruntfile.js

```diff
@@ -199,8 +199,8 @@ module.exports = function (grunt) {
           files: ['Gruntfile.js'],
           tasks: ['eslint']
         },
-        js: {
-          files: ['Gruntfile.js', 'src/**/*.js'],
+        all: {
+          files: ['Gruntfile.js', 'src/**/*.js', 'src/**/*.html', 'styles/**/*.css'],
           tasks: ['build']
```

Comment:
How do you feel about making this a separate PR since it is distinct from the actual feature work.